src/utils/tools/updatingIcons/prepareTheUpdate.ts

```diff
@@ -1,25 +1,31 @@
+/* eslint-disable dot-notation */
 import { IconInterface } from 'components/icons'
 
-const prepareUpdatedIcon = (newIconDetails: IconInterface) => {
+const prepareUpdatedIcon = (newIconDetails: IconInterface, currentDBicon: IconInterface) => {
   const updateDetails = {
     $set: {},
     $addToSet: {}
   }
   // loop through the keys
   for (const [key, value] of Object.entries(newIconDetails)) {
-    if (key !== 'name' && key !== 'tags' && key !== 'category') {
-      updateDetails.$set[key] = value
-    }
-    if (key === 'tags') {
+    if (key === 'svg' || key === 'svgOutlined') {
+      const isModified = newIconDetails[key] !== currentDBicon[key]
```

Comment:
so, if I understand correctly (considering that it is the first time I see this code), newIconDetails[key] contains the SVG information right? so we're comparing the SVG string from the DB with the one pulling now.
is this correct?

If this is the case, we will need to create the `lookModifiedAt` property according to the `key` so that we account for different themes

Reply:
Exactly, this's the case. I will commit the requested changes in minutes

Reply:
@cyntss Changes are committed and tested. Now, only the updated key will update its corresponding modifiedAt key.